server/controllers/apply/applicationsController.ts

```diff
@@ -145,12 +145,13 @@ export default class ApplicationsController {
           })
         }
       }
-
+      const restrictedPerson = req.flash('restrictedPerson')[0]
```

Comment:
could this error if req.flash doesn't have an entry for 'restrictedPerson'?

Reply:
gets undefined